apps/web/src/pages/index.tsx

```diff
@@ -244,24 +233,147 @@ const Home: NextPage = () => {
             }
             emptyState="No transactions"
           >
-            {latestTxsLoading ||
-            !latestTransactions ||
-            latestTransactions.length ? (
+            <div className="h-[630px]">
+              {latestBlocksLoading ? (
+                <div className="flex flex-col gap-3">
+                  {Array(LATEST_ITEMS_LENGTH)
+                    .fill(0)
+                    .map((_, i) => (
+                      <div className={CARD_HEIGHT} key={i}>
+                        <BlobTransactionCard compact />
+                      </div>
+                    ))}
+                </div>
+              ) : (
+                <SlidableList
+                  items={transactions.map((tx) => ({
+                    id: tx.hash,
+                    element: (
+                      <div className={CARD_HEIGHT} key={tx.hash}>
+                        <BlobTransactionCard
+                          transaction={{
+                            from: tx.from,
+                            to: tx.to,
+                            hash: tx.hash,
+                            rollup: tx.rollup,
+                            blobGasBaseFee: tx.blobGasBaseFee,
+                            blobGasMaxFee: tx.blobGasMaxFee,
+                          }}
+                          blobs={tx.blobs}
+                          compact
+                        />
+                      </div>
+                    ),
+                  }))}
+                />
+              )}
+            </div>
+          </Card>
+          <Card
+            header={
+              <div className="flex items-center justify-between gap-5">
+                <div>Latest Blobs</div>
+                <Button
+                  variant="outline"
+                  label="View All Blobs"
+                  onClick={() => void router.push(buildBlobsRoute())}
+                />
+              </div>
+            }
+            emptyState="No blobs"
+          >
+            <div className="h-[650px] sm:h-[630px]">
+              {latestBlocksLoading ? (
+                <div className="flex flex-col gap-3">
+                  {Array(LATEST_ITEMS_LENGTH)
+                    .fill(0)
+                    .map((_, i) => (
+                      <div className={CARD_HEIGHT} key={i}>
+                        <BlobTransactionCard compact />
+                      </div>
+                    ))}
+                </div>
+              ) : (
+                <SlidableList
+                  items={blobs.map((b) => ({
+                    id: b.versionedHash,
+                    element: (
+                      <div className={CARD_HEIGHT} key={b.versionedHash}>
+                        <BlobCard blob={b} transactions={[b.tx]} compact />
+                      </div>
+                    ),
+                  }))}
+                />
+              )}
+            </div>
+          </Card>
+          {/* <Card
```

Comment:
What do you think of just remove this now?

Reply:
Oops. Yes, I completely forgot about this.